client/my-sites/stats/post-performance/style.scss

```diff
@@ -0,0 +1,7 @@
+.stats-post-performance__start-post {
+	padding-bottom: 26px;
+}
+
+.stats-post-performance__no-posts-message {
+	margin-bottom: 1em;
```

Comment:
Is there a reason why this is using `em` and the other one is in `px`? Perhaps just use `px` for both for consistency? I think `14px` will do.